server/server.go

```diff
@@ -17,9 +17,9 @@ import (
 
 	"github.com/gobuffalo/packr"
 	golang_proto "github.com/golang/protobuf/proto"
-	grpc_middleware "github.com/grpc-ecosystem/go-grpc-middleware"
-	grpc_auth "github.com/grpc-ecosystem/go-grpc-middleware/auth"
-	grpc_logrus "github.com/grpc-ecosystem/go-grpc-middleware/logging/logrus"
+	"github.com/grpc-ecosystem/go-grpc-middleware"
+	"github.com/grpc-ecosystem/go-grpc-middleware/auth"
+	"github.com/grpc-ecosystem/go-grpc-middleware/logging/logrus"
```

Comment:
VSCode actually added all those aliases. Did your intellij remove it?

Reply:
Yes. intellij removed alias because `github.com/grpc-ecosystem/go-grpc-middleware/logging/logrus` has package `grpc_logrus` so alias technically is not required. I've added it back because otherwise, it is unclear without ide what `grpc_logrus` is